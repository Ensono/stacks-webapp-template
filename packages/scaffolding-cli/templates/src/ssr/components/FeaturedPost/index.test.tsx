import {render} from "@testing-library/react"
import React from "react"
import HeroPost from "."

test("renders HeroPost snapshot", () => {
    const {asFragment} = render(
        <HeroPost
            title=""
            coverImage=""
            date="01 Jan 1970 00:00:00 GMT"
            author=""
            excerpt=""
            slug=""
        />,
    )
    expect(asFragment()).toMatchSnapshot()
})
