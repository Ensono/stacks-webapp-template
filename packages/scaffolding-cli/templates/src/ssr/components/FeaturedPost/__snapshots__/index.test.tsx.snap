// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HeroPost snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
  >
    <button
      class="MuiButtonBase-root MuiCardActionArea-root"
      tabindex="0"
      type="button"
    >
      <div
        class="MuiPaper-root MuiCard-root makeStyles-card-1 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          class="makeStyles-cardDetails-2"
        >
          <div
            class="MuiCardContent-root"
          >
            <h2
              class="MuiTypography-root MuiTypography-h5"
            />
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-colorTextSecondary"
            >
              <time
                datetime="01 Jan 1970 00:00:00 GMT"
              >
                01-01-1970
              </time>
            </h6>
            <p
              class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-paragraph"
            />
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-colorPrimary"
            >
              Continue reading...
            </h6>
          </div>
        </div>
      </div>
      <span
        class="MuiCardActionArea-focusHighlight"
      />
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
</DocumentFragment>
`;
